articles/scale_on_aws.rst

```diff
@@ -0,0 +1,288 @@
+elastically add more machines to your cluster.
+
+Other operational benefits show themselves with high read/write
+throughputs. For example, you may need your application to talk to
+multiple endpoints instead of just one. In this case, Citus Cloud can
+set up DNS in front of your database instances and configure
+`PgBouncer <https://pgbouncer.github.io/>`__ to make sure that your
+distributed database stays within reasonable connection limits. The
+following shows a simplified architecture diagram.
+
+.. image:: ../images/citus-cloud-horizontal-scale.png
+
+5. Automatic Backups for Disaster Recovery
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your relational database powers your core business, and acts as your
+system of record, you need to take regular backups and store those
+backups in a highly durable location. This way, when the unexpected
+happens, you can resume your business and not lose it. Therefore, an
+integrated solution for disaster recovery is among the top feature
+requests prospective Citus customers ask for.
+
+The thing about database backups is that they are hard to get right.
+Fortunately, PostgreSQL has a rich ecosystem and comes with open source
+technologies for automatic backups. For example,
+`wal-e <https://github.com/wal-e/wal-e>`__ encrypts and continuously
+archives your data to a durable storage service, such as S3. In other
+words, wal-e make sure that your backup solution does the right thing,
+and cloud service providers make sure that your backups don’t get lost.
+
+.. image:: ../images/citus-cloud-auto-backup.png
+
+For Citus backups, wal-e helps with part of the picture. Taking
+distributed database backups are even harder. In this case, it helps
+when the author of wal-e also happens to be part of the Citus Cloud
+team. This way, when you deploy a Citus Cloud formation, we can
+automatically set up regular backups for your distributed database.
+
+6. Monitoring, Alerts, and Logging
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+An important part of running a relational database in production
+includes monitoring your database, alerting on anomalies, and setting up
+a unified logging infrastructure. This seems simple at first, but
+setting up this infrastructure usually takes time to set up and operate.
+
+At a minimum, you need to decide on the type of metrics you’d like to
+track. Some common ones include hardware metrics (such as CPU, memory,
+network), OS level statistics (VM, RSS, page faults), PostgreSQL metrics
+(pg\_stat views, table and index hit ratios), and active and long
+running queries.
+
+Once you decide on these metrics, you need to have a daemon that
+periodically collects these metrics, a solution to serialize and store
+these metrics, and a visualization layer to display them. A common stack
+that accomplishes this can run collectd on database instances, Graphite
+to store them, and Grafana to display them.
+
+.. image:: ../images/citus-cloud-monitoring.png
```

Comment:
This image looks too big. Could we make it smaller?

Reply:
(Done)